updatehub-sdk/src/client.rs

```diff
@@ -6,6 +6,18 @@ use crate::{api, Error, Result};
 use std::path::Path;
 use surf::{Body, StatusCode};
 
+/// The `Client` allow for requests to be sent.
+///
+/// This module is used to communicate with a running instance of the UpdateHub
+/// Agent using the following API methods:
+///
+/// - [abort_download](struct.Client.html#method.abort_download)
+/// - [info](struct.Client.html#method.info)
+/// - [local_install](struct.Client.html#method.local_install)
+/// - [log](struct.Client.html#method.log)
+/// - [probe](struct.Client.html#method.probe)
+/// - [remote_install](struct.Client.html#method.remote_install)
+
```

Comment:
I think this comments are more suited to a module level documentation (or crate level since Client is in the crate's root). Also link to any language construct can follow a language-linke notation, so you can use something like:
```Rust
/// - [abort_download](Client::abort_download)
```

This should make the comments more clean for anyone not reading in the HTML page while generating the same doc output.

Reply:
Make total senses, i'll change that.